src/compute-client/src/protocol/command.rs

```diff
@@ -366,6 +366,8 @@ pub struct ComputeParameters {
     pub persist: PersistParameters,
     /// Tracing configuration.
     pub tracing: TracingParameters,
+    /// Enable arrangement size logging
+    pub enable_arrangement_size_logging: Option<bool>,
```

Comment:
Nit: Could we move that below/above `enable_mz_join_core`, to keep the feature flags close together?